src/vfs.c

```diff
@@ -13,6 +18,7 @@
 #include "lib/assert.h"
 #include "lib/byte.h"
 
+#include "db.h"
```

Comment:
I probably didn't look enough to the diff, but why do we need to include ```d.h```?

Reply:
it's an artifact of some experimentation I think, removed.